### DISCUSSION TITLE
应用商城中应用更新问题

### DISCUSSION THREAD

**Discussion by LyingDoc** [1Panel-dev/1Panel#5592] [Q&A]
### 1Panel 版本

1.1.11

### 问题描述

我更新其中里升级应用，显示再升级中，过一段时间没有升级成功还是存在升级列表，也没报升级错误

### 重现步骤

![图片](https://github.com/1Panel-dev/1Panel/assets/131896721/629a0c8e-41b0-4f53-b656-b0fb66aa512c)
升级中，，，，
![图片](https://github.com/1Panel-dev/1Panel/assets/131896721/24709eea-b7b7-41b6-98f2-fe74c87ae7c3)
升级后

**Comment by wanghe-fit2cloud** [comment]
应用升级失败会使用备份自动回滚，可以进入 `日志审计` - `系统日志` 页面中查看失败原因。
> 如果是镜像拉取失败，可以参考：https://bbs.fit2cloud.com/t/topic/5886/
